enos/enos-scenario-e2e-docker-base-with-vault.hcl

```diff
@@ -52,7 +57,7 @@ scenario "e2e_docker_base_with_vault" {
     ]
     variables {
       image_name   = "${var.docker_mirror}/library/postgres:latest"
-      network_name = step.create_docker_network.network_name
+      network_name = [local.network_cluster]
```

Comment:
This is a little clunky, but it's due to a limitation of enos. You currently can't use step output variables in expressions, so I had to save the network name as a local variable.